bin/setup

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
```

Comment:
What is this file for?

Reply:
according to the internet, the setup is where you'll put scripts to set up the project for the first time when you start collaborating with others. detail: https://robots.thoughtbot.com/bin-setup